lib/main.coffee

```diff
@@ -6,6 +6,11 @@ SelectionCountView = require './selection-count-view'
 GitView = require './git-view'
 
 module.exports =
+  config:
+    cursorPositionFormat:
+      type: 'string'
+      default: '%L:%C'
```

Comment:
Although it's pretty self-explanatory, do you think it might be worth adding some help text here explaining what `%L` and `%C` are?